easybuild/main.py

```diff
@@ -208,7 +233,8 @@ def main(args=None, logfile=None, do_build=None, testing=False):
 
     # initialise logging for main
     global _log
-    _log, logfile = init_logging(logfile, logtostdout=options.logtostdout, silent=testing or options.terse)
+    _log, logfile = init_logging(logfile, logtostdout=options.logtostdout,
+                                 silent=(testing or options.terse)), color=options.color)
```

Comment:
(Kenneth Hoste, Fri, Mar 25, 2016 at 04:07:15PM -0700:)
> > @@ -208,7 +233,8 @@ def main(args=None, logfile=None, do_build=None, testing=False):
> >
> >      # initialise logging for main
> >      global _log
> > -    _log, logfile = init_logging(logfile, logtostdout=options.logtostdout, silent=testing or options.terse)
> > +    _log, logfile = init_logging(logfile, logtostdout=options.logtostdout,
> > +                                 silent=(testing or options.terse)), color=options.color)
>
> the `))` here should be `)`, syntax error
>
> @riccardomurri you didn't test this, did you? Did you?! ;-)

Damn Jenkins, you caught me!